### PR TITLE
Establish basic links between server and db

### PR DESCRIPTION
**Pull request by elocowan**
Make post route that successfully sends dummy data to db. Make get route that successfully sends reviews from db through server to client.

### CODE REVIEW

File: queries.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+const { Pool, Client } = require('pg')
+const connectionString = 'postgresql://localhost:5432/reviewsdb?user=ethan&password=ethan';
```
Comment (AngieD8615) [inline queries.js:2]: if this is a real user name and password, do you want it on github?

File: queries.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+const pool = new Pool({
+  connectionString: connectionString,
+})
+
+const getReviews = (res) => {
+  // add WHERE to retrieve reviews per product_id
+  // when dealing with pagination and count, add LIMIT
+  // when testing performance, take limit off to see what happens when asking for all from the set
```
Comment (AngieD8615) [inline queries.js:11]: I really like these comments.
You have not yet dealt with the pagination or count, right?

File: server.js [hunk 3]
```diff
@@ -17,84 +19,10 @@ app.use('/:productId', express.static(path.join(__dirname, "./client/dist")));
 app.get('/reviews/:product_id/list', (req, res) => {
   // console.log("req.params in list route", req.params);
   // console.log("req.query in list route: ", req.query);
```
Comment (AngieD8615) [inline server.js:21]: when you are ready, you should remove these console.logs

File: server.js [hunk 4]
```diff
@@ -17,84 +19,10 @@ app.use('/:productId', express.static(path.join(__dirname, "./client/dist")));
...
-        "reviewer_name": "new review ",
-        "helpfulness": 0,
-        "photos": []
-      }
-    ]
-  })
-})
+  getReviews()
+    .then((data) => res.send(data))
+    .catch((err) => res.err(err));
```
Comment (AngieD8615) [inline server.js:24]: do you also want to send a status code?

File: server.js [hunk 5]
```diff
@@ -138,7 +66,21 @@ app.get("/reviews/:product_id/meta", (req, res) => {
 // Adds a review for the given product
 app.post("/reviews/:product_id", (req, res) => {
   // console.log("req.body using bodyparser on review being added: ", req.body);
```
Comment (AngieD8615) [inline server.js:68]: when you are ready, be sure to remove the console.log

File: server.js [hunk 6]
```diff
@@ -138,7 +66,21 @@ app.get("/reviews/:product_id/meta", (req, res) => {
...
-  res.sendStatus(201);
+  const dummyReview = {
+      product_id: '1',
+      rating: 4,
+      date: '2020-08-28',
+      summary: 'okay product',
+      body: 'this was actually a so-so product',
```
Comment (AngieD8615) [inline server.js:74]: be sure to have more than 50 chars. The front end requires it. I know I was thinking about only displaying reviews that met this criterion.

**Review by AngieD8615 — APPROVED** [review]

File: queries.js (hunk 2 above)
Comment (elocowan) [reply queries.js:11]: no, I haven't dealt with those yet, but I'll be happy to have these notes when I do :)

**Pull request merged**